cibyl/models/ci/job.py

```diff
@@ -66,6 +66,16 @@ def __str__(self, indent=0, verbosity=0):
         if self.builds.value:
             for build in self.builds.values():
                 job_str += f"\n{build.__str__(indent+2, verbosity)}"
+        for attribute_name in self.plugin_attributes:
+            attribute = getattr(self, attribute_name)
+            if not attribute.value:
+                continue
+            if isinstance(attribute, AttributeValue):
+                job_str += f"\n{attribute.value.__str__(indent+2, verbosity)}"
+            else:
+                for attr_value in attribute.value:
+                    job_str += f"\n{attr_value.__str__(indent+2, verbosity)}"
+
```

Comment:
Nice! This is so clean. I would even put it in Model class __str__ together with models' attributes, but maybe in a separate PR